docs/sources/mimir/get-started/play-with-grafana-mimir/index.md

```diff
@@ -24,8 +24,22 @@ keywords:
 menuTitle: Play with Mimir
 title: Play with Mimir
 weight: 10
+killercoda:
+  title: Play with Mimir
+  description: Learn about Grafana Mimir, which provides distributed, horizontally scalable, and highly available long term storage for Prometheus.
+  details:
+    intro:
+      foreground: "docker-compose-update.sh"
+  preprocessing:
+    substitutions:
+      - regexp: 'tutorial-architecture.png'
```

Comment:
I think this should be fine as the linter wants, I think the issue with parsing must have been something else.

I've also escaped the period in the regular expression because we want to match it literally rather than match any character. I believe with YAML double quoted strings we need to escape the backslash

```suggestion
      - regexp: "tutorial-architecture\\.png"
```